src/views/IncorporationAgreement.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+  <div>
+    <section class="mt-10">
+      <header id="share-structure-header">
+        <h2>1. Incorporation Agreement and Articles</h2>
+      </header>
+      <p>
+        Before submitting your incorporation application you <b>must complete, sign, and date</b> an
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> Incorporation Agreement</span>
+          </template>
+          <span>
+            A document that is signed and dated by the people who agree to form the corporation. They are the
+            incorporators and the first shareholders of the corporation.
+          </span>
+        </v-tooltip>
+        , and a set of
+        <v-tooltip top color="primary">
+          <template v-slot:activator="{ on }">
+            <span v-on="on" class="tool-tip"> Benefit Company Articles</span>
+          </template>
+          <span>
+            The Articles for a benefit company must state the benefits the company intends to provide to society, as
```

Comment:
Check with Scott or Linda... I think this should be "Benefit Company" (proper noun).

Reply:
Also on line 35 below.

Reply:
And "articles" should maybe be lower case.

Reply:
All these texts are taken as such from the design and wireframes. I will ask Scott to go through those again. I will hold off those changes till I get a confirmation from them

Reply:
All entity type names are in lower case

Reply:
Update. For consistency, for this release , it can be kept as Benefit Company. After that it needs to be changed to benefit company